explorer/server/.env

```
@@ -36,6 +36,9 @@ STATIC_ROOT=../../ui/build
 # In testing we can point to grpcwebproxy (started in `hack/docker`) which is configured to allow CORS.
 UI_GRPC_URL=http://localhost:8401
 
+# The link to graphql playground, used in Home page of UI.
+UI_GRAPHQL_URL=http://devnet-graphql.casperlabs.io:40403/graphql
```

I think this `.env` file is only used in development mode, when the server is started in the console. Therefore it would make more sense to set this to the local node rather than devnet.
```suggestion
UI_GRAPHQL_URL=http://localhost:40403/graphql
```

---

The default setting to be used in the packaged app should be set in the [Dockerfile](https://github.com/CasperLabs/CasperLabs/blob/dev/explorer/Dockerfile).

---

Good suggestion. Done.